### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wanda-carlson**
Anna testing core features.

### CODE REVIEW

**Comment by wanda-carlson** [comment]
**Bert Griffin** pls check

**Review by bert-griffin — CHANGES_REQUESTED** [review]
@wanda-carlson requested changes 111

**Review by wanda-carlson — COMMENTED** [review]
ok

**Comment by wanda-carlson** [comment]
hi

**Comment by wanda-carlson** [comment]
delete

**Comment by chad-daniels** [comment]
delete

**Review by chad-daniels — COMMENTED** [review]
Screenshot (11)
Changes requested

**Review by wanda-carlson — COMMENTED** [review]
resolve testing

File: README.md [hunk 1]
```diff
@@ -8,3 +8,7 @@ Testing June 16.
 # Test June 21 12354647411111111111111111111111111111111111111111111111111111111111111111111111111
 This is a public repository managed by PaidPaper. The code is based in Python. It has several GitHub actions.
 Hi
+Testing core features June 29
+bbbbb new
+New New
+
```
Comment (wanda-carlson) [inline README.md:14]: resolve

**Pull request closed without merging**